bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/handler/PlugwiseHAApplianceHandler.java

```diff
@@ -146,10 +146,8 @@ protected void handleCommand(Appliance entity, ChannelUID channelUID, Command co
                 }
                 break;
             case APPLIANCE_OFFSET_CHANNEL:
-                if (command instanceof QuantityType quantityCommand) {
-                    Unit<Temperature> unit = entity.getOffsetTemperatureUnit().orElse(UNIT_CELSIUS).equals(UNIT_CELSIUS)
-                            ? SIUnits.CELSIUS
-                            : ImperialUnits.FAHRENHEIT;
+                if (command instanceof QuantityType<?> quantityCommand) {
+                    Unit<Temperature> unit = getRemoteTemperatureUnit(entity);
```

Comment:
`entity.getOffsetTemperatureUnit()` was previously considered, now it is `entity.getDHWTempUnit()`.
Isn't it a problem ?

Same remark for all next changes with `entity.getSetpointTemperatureUnit()`, `entity.getTemperatureUnit()`, ...

Reply:
The used unit is a global setting, so if one changes it from F to C it will change everywhere. That is why this doesn't matter and i chose to consolidate the determination in one method.`getRemoteTemperatureUnit`

Reply:
Ok